tests/stress_tests/memcheck_tests/tests.cpp

```diff
@@ -106,3 +106,54 @@ INSTANTIATE_TEST_CASE_P(MemCheckTests, MemCheckTestSuite,
                         ::testing::ValuesIn(
                                 generateTestsParams({"devices", "models"})),
                         getTestCaseName);
+
+TEST_P(MemCheckTestSuite, inference_with_streams) {
+    const auto nstreams = 2;
+    log_info("Inference of InferRequest from network: \"" << model
+                                                          << "\" with precision: \"" << precision
+                                                          << "\" for device: \"" << device << "\""
+                                                          << "\" with streams: \"" << nstreams);
+    auto test_pipeline = [&] {
+        MemCheckPipeline memCheckPipeline;
+
+        Core ie;
+        ie.GetVersions(device);
+        std::map<std::string, std::string> config;
```

Comment:
please, move helper objects definition and initialization in the top of lambda (means code 121 to 127)